tox.ini

```diff
@@ -41,6 +41,9 @@ commands =
 [testenv:common_logic]
 deps =
     -r{toxinidir}/f/common_logic/db_transformations.script.lock
+    -r{toxinidir}/f/common_logic/db_operations.script.lock
+    -r{toxinidir}/f/common_logic/save_disk.script.lock
+    -r{toxinidir}/f/common_logic/data_validation.script.lock
```

Comment:
if these scripts end up having competing requirements, we could have a problem. For now I'm ok though.